.gitignore

```diff
@@ -0,0 +1 @@
+*.sublime-*
```

Comment:
Please remove this as it is developer specific (and thus should go in the global gitignore on your machine) and not project specific (where anyone working on the project needs it)